stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/ReactionsView.kt

```diff
@@ -0,0 +1,80 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import android.content.Context
+import android.graphics.Canvas
+import android.util.AttributeSet
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.client.models.Reaction
+
+public abstract class ReactionsView : RecyclerView {
+
+    protected lateinit var reactionsViewStyle: ReactionsViewStyle
+
+    private lateinit var reactionsAdapter: ReactionsAdapter
+    private var isMyMessage: Boolean = false
+    private var reactionClickListener: ReactionClickListener? = null
+
+    public constructor(context: Context) : super(context) {
+        init()
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init()
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init()
+    }
```

Comment:
We could probably make these ctors `internal`, if we don't want people trying to create their own implementations of this class. The two provided public implementations should probably be enough.